---
layout: post
title: Moorea E5 Apulchra Metabolism Baseline Sampling Protocol
date: '2022-09-30'
categories: Protocols
tags: protocol, sampling
projects: E5, Putnam Lab
---

# Baseline Measurements Sampling Protocol - E5 *A. pulchra* metabolism project   

Original: 20220930   
Last Revised: 20220930 
Contact Ariana Huffmyer at ashuffmyer@uri.edu with any questions.   

# Overivew  

This protocol details the process for conducting baseline sampling for the E5 *A. pulchra* metabolism project in Mo'orea in Oct-Dec 2022. Proceedures for collecting and sampling biopsies are modified from D. Becker-Polinski same day processing protocol for *A. pulchra* which can be [found here](https://github.com/daniellembecker/Gametogenesis/blob/main/protocols/2021-12-26-Sample_Same_Day_Processing_Protocol.md).   

This protocol details sampling of n=4 fragments per genotype for RNA and DNA, epigenetics (methylation and chromatin structure), metabolomics, and physiology (e.g., symbiont density, biomass).  

The purpose of baseline sampling is to characterize response metrics at the level of each genotype prior to nutritional treatments. 

The response metrics are as follows:  

- DNA for 16S and ITS2 microbial and symbiont communities
- Metabolomics
- Chromatin structure  
- Gene expression  

For this protocol, we recommend processing n=4 fragments at a time (i.e., process all of one genotype at a time before moving to the next set). This will ensure that all genotypes are removed from the tanks for the same amount of time prior to sampling.    

# Materials required  

- 1 camera
- 1 ruler with size markings and red, green, and blue tape as color standards 
- 1 shallow bin with water proof paper at the bottom for photographing corals  
- 1 small cooler 
- Dry shipper 
- -40C freezer
- Liquid nitrogen 
- 96 1.5mL or 2mL screw cap cryo tubes 
- Stainless steel bone cutters 
- Paper towels
- Sharpies 
- 24 whirlpacks 
- 48 mL RNA/DNA shield 
- P1000 pipette and tips 
- 10% bleach
- 70% ethanol
- DI water 
- Gloves 
- 2 tube boxes 
- Tube racks 
- Alcochol wipes 

Set up all equipment in the molecular lab for processing.  

# Datasheets required  

Open the following data sheets in preparation for sampling on your computer (requires cloning the repository onto your computer using Git in the command line):    

- [Fragment tracking and metadata](https://github.com/urol-e5/apulchra_metabolism/blob/main/data/fragments.csv)
- [Baseline sampling checklist](https://github.com/urol-e5/apulchra_metabolism/blob/main/data/baseline_sampling/baseline_sampling_checklist.csv)
- [Sample inventory list](https://github.com/urol-e5/apulchra_metabolism/blob/main/data/baseline_sampling/sample.id.csv)

# Preparing tubes and whirlpacks    

Label whirlpacks with fragment number (n=24 whirlpacks) and project information. An exmaple is as follows:  

```
E5 Apul Metabolism
Oct 2022
Baseline
A7
Putnam 
```

Label tubes in consective numbers on the top cap (#1-96). On the side of the tube, label with `E5 Metab. 2022` and the tube number again.  

Fill tubes #1-48 with 1 mL of RNA DNA shield each. Tubes 49-96 will be used to freeze in liquid nitrogen. Record tube numbers on the [Sample inventory list](https://github.com/urol-e5/apulchra_metabolism/blob/main/data/baseline_sampling/sample.id.csv).  

Label two tube boxes with the following information:  

```
E5 A. pulchra Metabolism
Putnam 
October 2022 
Baseline 
Samples in RNA DNA Shield
Box #1
```

```
E5 A. pulchra Metabolism
Putnam 
October 2022 
Baseline 
Samples frozen with lN2
Box #2
```

# 1. Fragment photographs  

First, take a photograph of each fragment for reference of pigmentation and size using the following proceedure:  

- Remove fragments that need to be sampled of one genotype at a time to do all processing steps below before moving onto the next fragment (n=4 fragments of A, B, C, D, E, F in turn). Place fragments in a small cooler filled with seawater. Each time fragments are removed for processing, record the date sampled on the [fragment tracking and metadata sheet](https://github.com/urol-e5/apulchra_metabolism/blob/main/data/fragments.csv).  
- Remove/clean algae from string and coral fragment.  
- Place the fragment at the bottom of a shallow bin on top of waterproof paper. Place a ruler with color standard (R, G, B) next to the fragment. Place the clip with fragment label in the frame of the photograph.
- Take a picture and 
- After all photographs are completed for the day, save the photos in the E5 [apulchra_metabolism GitHub repo in the fragmetn_photos folder here](https://github.com/urol-e5/apulchra_metabolism/tree/main/data). 
- Add "yes" to the photograph column in the [Baseline sampling checklist](https://github.com/urol-e5/apulchra_metabolism/blob/main/data/baseline_sampling/baseline_sampling_checklist.csv) after taking a photo of each fragment. Ensure that all fragments have a photograph.  
- After all photographs are taken at the end of the day, rinse the bin with fresh water and store for the next sampling.  

# 2. Sampling biopsies in the lab  

For each fragment (process one genotype at a time, n=4 fragments per genotype) process with the following proceedure:  

## Prepare supplies  

- Wear gloves during the entire processing process.  
- Add some liquid nitrogen to a small cooler. 
- Place a clean paper towel on the bench top. 
- Prepare the boxes, tubes, and whirlpack needed. For each fragment, look at the tube numbers identified on the [Sample inventory list](https://github.com/urol-e5/apulchra_metabolism/blob/main/data/baseline_sampling/sample.id.csv).  
- There will be n=2 RNA DNA shield tubes and n=2 snap freeze tubes per fragment. There will be n=1 whirlpack per fragment.  
- Clean the bone cutters by first immersing in 10% bleach, then 70% ethanol, then DI water.  

## Sampling   

- First, sample biopsies for the tubes that need to be snap frozen. These will be used for metabolomics and chromatin structure. 
- **It is critical to wear gloves and not allow contamination between samples. We are measuring sensitive metrics and contamination prevention is critical.**  
- Use the bone cutters to scrap a 5-10 polyp area off the base - mid branch of the fragment. Scrape this into the first snap freeze tube. Repeat this for the second tube.  
- Immediately freeze these two tubes in liquid nitrogen. 
- Next, use the bone cutters to scrap a 5-10 polyp area off the base of the fragment. Put into the first DNA RNA shield tube. Repeat this for the second tube.  
- Put the RNA DNA shield tubes into the respective box in the -40C freezer. 
- Remove the tubes from liquid nitrogen, put in the respective box and into the -40C freezer. 
- Take the remaining fragment and put into the whirlpack bag. Close securely and place in the -40C freezer. 
- Immerse the bone cutters in 10% bleach, 70% ethanol, and DI water between fragments. 
- Wipe gloves with an alcohol wipe. 
- Replace the paper towel. 
- Repeat this process for all fragments.  
- Check off the respective columns for each fragment on the [baseline sampling checklist](https://github.com/urol-e5/apulchra_metabolism/blob/main/data/baseline_sampling/baseline_sampling_checklist.csv).  

# 3. Clean up and notebooks  

- Clean up all supplies and ensure all samples are in the freezer. 
- Update the [sample inventory list](https://github.com/urol-e5/apulchra_metabolism/blob/main/data/baseline_sampling/sample.id.csv) to include a location - box number and freezer - for all samples.  
- Ensure that the [fragment tracking and metadata](https://github.com/urol-e5/apulchra_metabolism/blob/main/data/fragments.csv) and [baseline sampling checklist](https://github.com/urol-e5/apulchra_metabolism/blob/main/data/baseline_sampling/baseline_sampling_checklist.csv) are completed.  
- Add a notebook entry to the [E5 notebook for this project](https://github.com/urol-e5/urol-e5.github.io/blob/master/_posts/2022-09-20-E5-Moorea-Adult-Stressor-Experiment.md).  



